drivers/base/bootvid/arm/bootvid.c

```diff
@@ -56,7 +56,8 @@ DisplayCharacter(IN CHAR Character,
 
 VOID
 NTAPI
-VgaScroll(IN ULONG Scroll)
+VidpScroll(
```

Comment:
```suggestion
Scroll(
```
to emphasise it's a per-arch dependent global helper function and not a generic private bootvid helper.